docs/examples/crawl_relative_links_all.ts

```diff
@@ -1,4 +1,7 @@
-import { CheerioCrawler, EnqueueStrategy } from '@crawlee/cheerio';
+import { CheerioCrawler, EnqueueStrategy } from 'crawlee';
```

Comment:
I find the `EnqueueStrategy` imports a bit too Java-eque. Let's skip them and use just the strings. It's common in Puppeteer and Playwright as well.